.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+# Description
+
+## What is the change being pushed?
+
+<!--
+ Describe **what** the change is. e.g.,:
+  - fixing a soundness bug in the the IsLessThan() template
+  - adding a new test for IsLessThan() to ensure that 3 < 3 cannot be satisfied
+  - adding an new `/vk` endpoint to the prover service
+-->
+
+## Why are you pushing this change?
+
+<!--
+ Describe **why** you are making this change. Give as much context as needed,
+ such as when/how a bug was found, why is it important that it be fixed, etc. 
+ e.g.,:
+  - IsLessThan() was underconstrained and wrongly allowed for 3 < 3. This would lead to a soundness break in higher level applications.
+  - Important for our tests cover all edge cases.
+  - We want to make it easy for anybody to see what VK the prover service is currently using.
+-->
+
+## How is this implemented?
+
+<!--
+ Describe **how** your change was implemented.
+ The goal is to help reviewers and future readers understand this PR. e.g.,:
+  - explain what well-known algorithms you are using, if any
+  - explain the structure of the code
```

Comment:
not sure what "structure of the code" means...

Reply:
e.g.,
 - this code implements an algorithm `lagrange_coeffs` for computing the Lagrange coefficients for an evaluation domain `S`.
 - `lagrange_coeffs` mainly relies on a few algorithms:
     + an FFT-based `poly_mul` algorithm for multiplying polynomials so as to derive an accumulator polynomial from its roots
     + a `poly_diff` algorithm for differentiating polynomials, applied on said accumulator polynomial
     + an `fft` algorithm for computing FFTs, applied on the derivative from above
     + finite field operations, especially over roots of unity

Not saying it has to be like this: just find the best way to help the reviewer understand the structure / scaffolding / architecture / layout / organization of your code, especially if it is a complex one.